src/cpp/src/llm_pipeline_static.cpp

```diff
@@ -372,7 +443,7 @@ EncodedResults StaticLLMPipeline::generate(
     m_prefill_request.infer();
 
     // NB: Now there are prompt_len tokens in KV-cache
-    m_kvcache_desc.num_stored_tokens += prompt_len;
+    m_kvcache_desc.num_stored_tokens += uint32_t(prompt_len);
```

Comment:
and here, maybe `prompt_len` can be in correct type

Reply:
The same as with offset, used `static_cast`